src/ui/mod.rs

```diff
@@ -1,5 +1,29 @@
+use std::path;
+
 mod tui_backend;
 pub mod views;
 pub mod widgets;
 
 pub use tui_backend::*;
+
+#[derive(Debug, Clone, Copy, Hash, PartialEq, Eq)]
+pub struct Rect {
```

Comment:
Is there a reason for making our own `Rect` instead of using https://docs.rs/tui/0.16.0/tui/layout/struct.Rect.html

Reply:
Yeah, lol, changed it two times back and forth. I went with this to stay more independent from the backend lib, as this would have been the only TUI structure which is used in the UI-independent logic. I can change that, if you like.

Reply:
No, its fine lol. Was just curious :+1: